Modules/Package.swift

```diff
@@ -54,14 +54,16 @@ let package = Package(
         .package(url: "https://github.com/wordpress-mobile/wpxmlrpc", from: "0.9.0"),
         .package(url: "https://github.com/wordpress-mobile/NSURL-IDN", revision: "b34794c9a3f32312e1593d4a3d120572afa0d010"),
         .package(url: "https://github.com/zendesk/support_sdk_ios", from: "8.0.3"),
-        .package(url: "https://github.com/wordpress-mobile/GutenbergKit", from: "0.10.1"),
+        .package(url: "https://github.com/wordpress-mobile/GutenbergKit", revision: "df607f9380891970d0d296010abf93d582fecdb0"),
```

Comment:
Friendly note (to self): we'll need to publish a GBK release and integrate it here before merging. I introduced this in https://github.com/wordpress-mobile/WordPress-iOS/pull/25005/commits/51e6c646f4f49e1863dc5d5c0490252bac433e2b.